### ISSUE TITLE
issues with node-test-commit-osx

### ISSUE THREAD

**Issue by panva** [nodejs/build#3116]
We're not getting any successful buils through on [node-test-commit-osx](https://ci.nodejs.org/job/node-test-commit-osx/) since at least 2 days ago.

<img width="335" alt="image" src="https://user-images.githubusercontent.com/241506/207868604-27a27999-3c17-4ea1-88a2-a98cfc78f378.png">

**Comment by panva** [comment]
cc @nodejs/build

**Comment by targos** [comment]
that's because all `test-orka-*` machines are offline. /cc @AshCripps

**Comment by AshCripps** [comment]
Welp macstadium have decided to update the cluster this week instead of january when I booked the timeslot in for.

**Comment by joyeecheung** [comment]
I actually got a green one but the results weren't communicated back to the parent job, so it gets stuck there: https://ci.nodejs.org/job/node-test-commit-osx/49256/

**Comment by UlisesGascon** [comment]
I will try to re-deploy the testing VMs

**Comment by richardlau** [comment]
> * Only the agent [test-orka-macos10.14-x64-1](https://ci.nodejs.org/computer/test-orka-macos10.14-x64-1/) was working for few hours but I relocated the vm as actually it was a Macos11.x due some NAT misleading configuration.

The [test-orka-macos10.14-x64-1](https://ci.nodejs.org/computer/test-orka-macos10.14-x64-1/) agent in Jenkins is on a macOS 10.14 host:
![image](https://user-images.githubusercontent.com/5445507/208097930-3779f148-0e81-455a-a54b-6852c18d32df.png)

**Comment by UlisesGascon** [comment]
At this point all the Orka machines are up and running. We keep working on Orka in a separate discussion #3112

**Comment by panva** [comment]
Big thank you to everyone involved!

**Comment by targos** [comment]
It seems that all macs are now back on Java 8. How is that?

**Comment by UlisesGascon** [comment]
Confirmed @targos, We have Java 8 in all the Orka machines (10.x and 11):

```
openjdk version "1.8.0_292"
OpenJDK Runtime Environment (AdoptOpenJDK)(build 1.8.0_292-b10)
OpenJDK 64-Bit Server VM (AdoptOpenJDK)(build 25.292-b10, mixed mode)
```

The current vms are restored from an old backup available in Orka and I had some issues running ansibe (https://github.com/nodejs/build/issues/3119). Yesterday I run Ansible successfully against the macos11 vms.

I just realized that I run an old version of the playbook that didn't include your changes in #3085. So, I will pull the last changes to my machine and re-run the Ansible playbooks. Maybe that will solve #3119 too
